apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -158,7 +138,6 @@ def read(
         * Negative indexing is unsupported.
         """
         del batch_size, partitions, platform_config  # Currently unused.
-        self._check_open_read()
```

Comment:
Is this related to the PR?

Reply:
This was something from another change that leaked into this one. Removing (un-removing?) it.